app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -51,14 +51,16 @@ class PatientSearchResultItemView(
     renderPatientNameAgeAndGender(searchQuery, model)
```

Comment:
Could we have made the last commit and this commit into one? Considering the last commit is where we removed the visibility of the `phoneNumberContainer`. WDYT?

Reply:
This commit only has the patient name highlighting. I can squash the phone number highlight and the visibility commit into one.

Reply:
Oh, I was not talking about this function. I was talking about the commit. I had to select somewhere to leave a comment so I selected the top of the file in this commit. and yeah let's combine these 2 commits.